src/api.ts

```diff
@@ -347,38 +544,12 @@ async function uncompositeStyle(
   return style
 }
 
-/**
- * Setup data storage paths and create Tilestore instances for existing mbtiles.
- * TODO: Watch dirs for user-added files? Maybe separate user dir from these
- * data dirs, to avoid users moving mbtiles internally managed here.
- */
-async function init(dataDir: string): Promise<Context> {
-  const paths = { tilesets: '', styles: '', sprites: '', fonts: '', db: '' }
-  for (const pathName of Object.keys(paths) as Array<keyof typeof paths>) {
-    paths[pathName] = path.join(process.cwd(), dataDir, pathName)
-    await mkdirp(paths[pathName])
-  }
-
-  const db = Level(paths.db)
-  const stylesDb = SubLevel<string, OfflineStyle>(db, 'styles')
-  const etagDb = SubLevel<string, string>(db, 'etag', {
-    valueEncoding: 'string',
-  })
-  const cacheDb = SubLevel<string, Buffer>(db, 'urlCache', {
-    valueEncoding: 'binary',
-  })
-  const swrCache = new SWRCache({ etagDb, cacheDb })
+function init(dataDir: string): Context {
+  const db = new Database(path.resolve(dataDir, 'mapeo-map-server.db'))
```

Comment:
didn't have the best idea of where we're embedding the db and what to call it. open to suggestions here